app/domain/authentication/authn_gcp/consts.rb

```diff
@@ -0,0 +1,8 @@
+
+module Authentication
+  module AuthnGcp
+
+    PROVIDER_URI = "https://accounts.google.com"
```

Comment:
@jonahx this consts is used in several places in the `AuthnGcp` module. is this the best practice to define it? creating a `consts.rb` file?